### ISSUE TITLE
Building on Linux

### ISSUE THREAD

**Issue by foreverWIP** [NatsumiFox/AMPS-Sonic-2#1]
It seems the build.sh hasn't been updated to match build.bat. I'm looking into it now; have you considered other ways of including the AMPS files? If a separate program is needed to properly figure out offsets I'm willing to assist with a Linux port of Includer.exe.

**Comment by Awuwunya** [comment]
Yes, unfortunately a separate program is needed. I am not quite sure how well Includer would be possible to be ran on Linux using mono (it's written in C#), but really a C port would be more easy for everyone. Trouble is, I am terrible at C. Also I do not know if ConvPCM or ConvSym would work on Linux either.

**Comment by foreverWIP** [comment]
Who said it had to be written in C or C#? Python comes standard in most Linux installs, maybe that would be a better fit. It's not like it's worse at manipulating individual bytes, sure there's more overhead than C, but it's still quite minimal anyway.

**Comment by Awuwunya** [comment]
Well I am worried about Python too because you'd have to install it on Windows and any Linux/Mac that doesn't have it. Granted, knowing the audience its likely no problem, but its still extra stuff to install. Same thing for C# too but its more likely already installed on Windows machines that are the only properly supported platform now anyway. I also never have programmed in Python anyway so that's another difficulty with it.

**Comment by RobiTheGit** [comment]
the AMPS/Data file is missing when i install it

**Comment by RobiTheGit** [comment]
Just use wine and run the build.bat script, that is how I do it
